addons/event/views/event_views.xml

```diff
@@ -554,7 +553,8 @@
                                     ]"
                                     attrs="{'invisible': [('event_id', '=', False)]}"
                                     options="{'no_open': True, 'no_create': True}"/>
-                                <field name="date_open" groups="base.group_no_one"/>
+                                <field name="partner_id"/>
```

Comment:
Why do we add partner_id here? It's already in the "Attendee" tab.
Also seems a bit unrelated to this commit.

Reply:
Accidently added to a wrong commit.